upa/src/tool/devAggregator.ts

```diff
@@ -20,6 +20,13 @@ import {
   packOffChainSubmissionMarkers,
 } from "../sdk/aggregatedProofParams";
 
+type SubmissionIntervalWithMetadata = SubmissionInterval<
+  | {
```

Comment:
Fine to add support here, but just to check - the dev-aggreagtor isn't yet able to read off-chain submissions, presumably?

Reply:
Yeah it currently reads only from on-chain events. But I added general support for off-chain handling. It defaults to `undefined` which is falsy.

We could add reading offchain submissions from file later